From 20a63f9622b6795790b0864cc5dcfd05f93b3ddd Mon Sep 17 00:00:00 2001
From: Jacky Chou <jacky_chou@aspeedtech.com>
Date: Tue, 5 May 2026 13:32:35 +0800
Subject: [PATCH] pinctrl: aspeed: extend AST2700 pinctrl to cover AST2705

AST2705 shares the AST2700 pin-controller, so extend the
ASPEED_AST2700_PINCTRL dependency to include CONFIG_ASPEED_AST2705.

Upstream-Status: Backport [SDK v11.03 migration]
Signed-off-by: Jacky Chou <jacky_chou@aspeedtech.com>
Change-Id: Ib30530c5140ce776ddc804701b7e6c4f9f665aa0
Signed-off-by: rupeshs <rupeshs@ami.com>
---
 drivers/pinctrl/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pinctrl/Kconfig b/drivers/pinctrl/Kconfig
index 26973b0ab6..1ed4e246df 100644
--- a/drivers/pinctrl/Kconfig
+++ b/drivers/pinctrl/Kconfig
@@ -327,7 +327,7 @@ config ASPEED_AST2600_PINCTRL
 
 config ASPEED_AST2700_PINCTRL
         bool "Aspeed AST2700 pin control driver"
-        depends on DM && PINCTRL_GENERIC && ASPEED_AST2700
+        depends on DM && PINCTRL_GENERIC && (ASPEED_AST2700 || ASPEED_AST2705)
         default y
         help
           Support pin multiplexing control on Aspeed ast2700 SoC. The driver
-- 
2.43.0

